docs/guide/modules.md

```diff
@@ -280,6 +280,8 @@ export function createPlugin (options = {}) {
 You can register a module **after** the store has been created with the `store.registerModule` method:
 
 ``` js
+import { store } from 'vuex'
```

Comment:
Instead of importing, can we add a code to create a store module so that the user can know what is needed before using `registerModule`?

```suggestion
import Vuex from 'vuex'

const store = new Vuex.Store({ /* options */ })
```

Reply:
Great idea